site2/docs/security-bouncy-castle.md

```diff
@@ -37,8 +37,7 @@ Pulsar module `bouncy-castle-bc`, which defined by `bouncy-castle/bc/pom.xml` co
     </dependency>
 ```
 
-And based on Pulsar module `bouncy-castle-bc`, Pulsar shades a fat jar in module `bouncy-castle-bc-shaded` that contains needed classes of BouncyCastle non-FIPS jars.
-By using this `bouncy-castle-bc-shaded` module, user can easily include and exclude BouncyCastle non-FIPS jars.
+By using this `bouncy-castle-bc` module, user can easily include and exclude BouncyCastle non-FIPS jars.
```

Comment:
```suggestion
By using this `bouncy-castle-bc` module, you can easily include and exclude BouncyCastle non-FIPS jars.
```

Use the second-person pronoun `you` as much as possible in technical writing.

Please check all occurrences.